src/Jobs/SendNotificationWhenPostIsFlagged.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace FoF\Subscribed\Jobs;
+
+use Flarum\Flags\Flag;
+use Flarum\Notification\NotificationSyncer;
+use Flarum\User\User;
+use Flarum\Queue\AbstractJob;
+use FoF\Subscribed\Blueprints\PostFlaggedBlueprint;
+use Illuminate\Database\Query\Expression;
+
+class SendNotificationWhenPostIsFlagged extends AbstractJob
+{
+    /**
+     * @var Flag
+     */
+    protected $flag;
+
+    public function __construct(Flag $flag)
+    {
+        $this->flag = $flag;
+    }
+    
+    public function handle(NotificationSyncer $notifications)
+    {
+        $post = $this->flag->post;
+        $flag = $this->flag;
+
+        $notify = User::query()
+            ->where('users.id', '!=', $flag->actor->id)
```

Comment:
Should this not be $flag->user ? rather than $flag->actor ?

The `user()` is the relationship, `actor` only happens to be there upon an authenticated request from the http kernel.

If this gets executed by a queue worker, the `actor` isn't populated, however `user` is as this is a defined relationship.

Reply:
That's a great spot! How did this ever work 🙈

Reply:
Oh wait this is new code, okay makes sense. Let's get it fixed @imorland can probably tag a release somewhere the coming days.